src/Experimental/Html.fs

```diff
@@ -116,20 +114,21 @@ let begin_element_to_string = function
        |> String.Concat
      sprintf "<%s %s>" e ats
 
-let end_element_to_string = function
+let internal endElementToString = function
```

Comment:
Yes, it would be good to do that. I started down that path but reverted some of it because I wasn't sure. Let's do that in a follow up PR